example_test.go

```diff
@@ -1080,13 +1083,30 @@ func ExampleSchema() {
 	// Space 2 ID 616 schematest
 }
 
+// Example demonstrates how to update the connection schema.
+func ExampleSchema_update() {
```

Comment:
To make the example connected to the update method.
```suggestion
// Example demonstrates how to update the connection schema.
func ExampleConnection_OverrideSchema() {
```

Reply:
Updated.